packages/adapter-drizzle-all/src/index.ts

```diff
@@ -0,0 +1,26 @@
+import { PgAdapter } from "./pg"
+import { MySqlAdapter } from "./mysql"
```

Comment:
should we get the other DB adapters from their packages?
```suggestion
import { MySqlAdapter } from "@next-auth/drizzle-adapter-mysql"
```